packages/@ama-sdk/core/package.json

```diff
@@ -41,6 +41,7 @@
     "uuid": "^8.3.2"
   },
   "peerDependencies": {
+    "@o3r/schematics": "workspace:^",
```

Comment:
Should be in `dep` in this case because we don't want to require `@o3r/schematics` as dep in the SDKs